command/core.py

```diff
@@ -0,0 +1,23 @@
+"""Calls the appropriate handler depending on the event data."""
+from command.commands.user import UserCommand
+
+
+class Core:
+    """Encapsulate methods for handling events."""
+
+    def __init__(self):
+        """Initialize the dictionary of command handlers."""
+        self.__commands = {}
+        self.__commands["user"] = UserCommand()
+
+    def handle_app_mention(self, event_data):
+        """Handle the events associated with mentions of @rocket."""
+        message = (event_data["event"])["text"]
+        s = message.split(' ', 2)
+        if s[0] != "@rocket":
+            return False
+        else:
+            command_type = s[1]
+            command = command_type + ' ' + s[2]
+            self.__commands[command_type].handle(command)
```

Comment:
Should add a fail-safe (try-catch) for when an invalid command is used.

Reply:
Ok I'll modify the function so it returns 0 on invalid mention, 1 on successful handle, and -1 on error. Is that ok?

Reply:
Eventually we'll want it to log as well, though since we never got logging set up...

Reply:
@rwblickhan is it a good idea to throw exceptions here? and to let whoever to handle them? as in logging?

Reply:
Personally I think the best solution here would be to not return/throw anything, but rather to just log an error and maybe message the user that sent it (something like "Invalid command"), but seeing as how we don't have logging or Slack messages set up yet, that doesn't really work.

(I think we would want to avoid doing error handling in the Slack events handling module that will call this, so that that can remain as simple as possible.)